src/main.cpp

```diff
@@ -349,9 +346,6 @@ int main(int argc, char *argv[])
       case 'u':
         safe_mode = false;
         break;
-      case 'b':
-        force_btf = true;
-        break;
```

Comment:
Can we make this a noop instead? A noop with this PR is the same behavior as before right? And we can't reuse '--btf` anyways so we might as well leave it in to not break existing invocations people might have saved.

Reply:
Yeah, that sounds reasonable, will do it.